src/data/static/testimonials.ts

```diff
@@ -0,0 +1,48 @@
+export const data = [
```

Comment:
export const data: ICardDetails = {}

- we can type the data too like I mentioned above
- Since it fits with the testimonials data. Does that interface have the right name?